internal/pkg/controllers/spod/bindata/spod.go

```diff
@@ -375,6 +423,15 @@ var Manifest = &appsv1.DaemonSet{
 							},
 						},
 					},
+					{
+						Name: "host-varsyslog-volume",
+						VolumeSource: v1.VolumeSource{
+							HostPath: &v1.HostPathVolumeSource{
+								Path: "/var/log/syslog",
```

Comment:
```suggestion
								Path: config.SyslogDropDirectory,
```

Reply:
Thank you, made the change considering the new target `/dev/kmsg`.